CppCustomVisualizer/README.md

```diff
@@ -0,0 +1,15 @@
+## C/C++ Custom Visualizer
+This directory contains an example C/C++ Custom Visualizer.
+
+## What is a Custom Visualizer?
+The Visual Studio C/C++ Expression Evaluator allows customization of the display of variables in
+the debugger based on the type of that variable (example: change the way CExampleClass variables
+are displayed) using .natvis files. Natvis files add rules to format a type either declaratively or
+by running custom code in the debugger. This custom code is called a Custom Visualizer. Most of
+the time as it easier and much less error prone to use the declarative approach. But
+Custom Visualizers are useful when formatting a type is too complicated to be done declaratively.
+
+Natvis documentation can be found on [docs.microsoft.com](https://docs.microsoft.com/en-us/visualstudio/debugger/create-custom-views-of-native-objects).
+
+## How to use this sample
+More information about this sample can be found in the [Wiki for this project](https://github.com/Microsoft/ConcordExtensibilitySamples/wiki/C-C---Custom-Visualizer-Sample).
```

Comment:
The link should change to `https://github.com/Microsoft/ConcordExtensibilitySamples/wiki/Cpp-Custom-Visualizer-Sample`. GitHub wiki's apparently don't like '/' or '+' in the title.